content/ansible-tower-getting-started/2-cred/_index.md

```diff
@@ -177,3 +178,43 @@ The yellow output of the command indicates Ansible has actually done something (
 {{% notice tip %}}
 Try to click one of the output lines in the window showing the job output. A small window with a lot of information about the job like module args will open. Have a look and close it again.
 {{% /notice %}}
+
+## Execution Environments: A deeper look
+
+As promised let's look a bit deeper into execution environments. During the section covering ad hoc commands you have already seen you have to choose an execution environment, the same will hold true for running Playbooks later on. In your Automation Controller web UI, go to **Administration → Execution Environments**. You'll see a list of the configured execution environments and original location of the image, in our case the images are provided in the **quay.io** container registry. Here you could add your own registry with custom EE images, too.
+
+So what happens, when Automation Controller runs an ad hoc command or Playbook? Let's see... first get prepared to run an ad hoc command again:
+
+- Go to **Views -> Jobs** and click one of your ad hoc command runs 
+- In the following view switch to the **Details** tab
+- Your are now prepared to restart the ad hoc command by clicking **Relaunch**
+
+You should already have your code-server terminal open in another browser tab, if not open https://{{< param "external_code" >}} and do **Terminal -> New Terminal** in code-server. In this terminal:
+
+- You should be your student user, become the **awx** user by running `sudo -u awx -i`
+- First let's look for the image:
+```
+[awx@ansible-1 ~]$ podman images
+REPOSITORY              TAG     IMAGE ID      CREATED       SIZE
+quay.io/ansible/awx-ee  0.2.0   68b8d8c4702d  2 months ago  1.25 GB
```

Comment:
well, it says "2 months ago" and I just ran my ad-hoc command a minute ago - the next sentence makes no sense